Pipfile

```diff
@@ -28,15 +27,12 @@ aioboto3 = "*"
 aiohttp = "*"
 gunicorn = "*"
 uvicorn = {extras = ["standard"], version = "*"}
-fastapi = "*"
+fastapi = {extras = ["standard"], version = "*"}
```

Comment:
Change in [fastapi 0.112.0](https://fastapi.tiangolo.com/release-notes/#01120) requires `fastapi[standard]` to install [extras](https://fastapi.tiangolo.com/#dependencies). This is needed because we use `TestClient` in our tests.